docs/source/gee.rst

```diff
@@ -121,6 +121,7 @@ The distribution families are the same as for GLM, currently implemented are
    InverseGaussian
    NegativeBinomial
    Poisson
+   Tweedie
```

Comment:
Is tweedie allowed in gee?
I guess there is nothing special enough that it shouldn't work

Reply:
I would think so. Granted, I'm no expert here.